<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>973</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);
border-color: rgb(255, 255, 255);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:700; text-decoration: underline; color:#aa0000;&quot;&gt;Realtime Garbage Monitor&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt; font-weight:700; color:#aa007f;&quot;&gt;HOUSE ID&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:700; color:#aa007f;&quot;&gt;Garbage Weight(Kgs)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:700; color:#aa007f;&quot;&gt;Garbage Volume(cm3)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:700; color:#aa007f;&quot;&gt;Status&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLCDNumber" name="lcdNumber">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
            </property>
            <property name="digitCount">
             <number>9</number>
            </property>
            <property name="mode">
             <enum>QLCDNumber::Bin</enum>
            </property>
            <property name="segmentStyle">
             <enum>QLCDNumber::Flat</enum>
            </property>
            <property name="intValue" stdset="0">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="lcdNumber_2">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
            </property>
            <property name="digitCount">
             <number>8</number>
            </property>
            <property name="segmentStyle">
             <enum>QLCDNumber::Flat</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="lcdNumber_3">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
            </property>
            <property name="digitCount">
             <number>8</number>
            </property>
            <property name="segmentStyle">
             <enum>QLCDNumber::Flat</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="h1status">
            <property name="font">
             <font>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;TextLabel&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="textFormat">
             <enum>Qt::AutoText</enum>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_4">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLCDNumber" name="lcdNumber_5">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
            </property>
            <property name="digitCount">
             <number>9</number>
            </property>
            <property name="segmentStyle">
             <enum>QLCDNumber::Flat</enum>
            </property>
            <property name="intValue" stdset="0">
             <number>2</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="lcdNumber_6">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
            </property>
            <property name="digitCount">
             <number>8</number>
            </property>
            <property name="segmentStyle">
             <enum>QLCDNumber::Flat</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="lcdNumber_7">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
            </property>
            <property name="digitCount">
             <number>8</number>
            </property>
            <property name="segmentStyle">
             <enum>QLCDNumber::Flat</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="h2status">
            <property name="font">
             <font>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;TextLabel&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="styleSheet">
          <string notr="true">color:rgb(170, 0, 127);</string>
         </property>
         <property name="text">
          <string>Refresh</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
